torch/_inductor/config.py

```diff
@@ -74,6 +74,9 @@
 # If not specified, a temp directory will be created under the default caching path
 aot_inductor_output_path = ""
 
+# TODO: Temporary flag: If we are passing from export, ignore aot autograd for now
+ignore_aot_autograd = False
```

Comment:
Let's move this and maybe `_in_aot_compilation` (my bad on that one) to `virtualized.py`, see: [this pr](https://github.com/pytorch/pytorch/pull/103289/files?show-viewed-files=true&file-filters%5B%5D=)  

adding `get_real_inputs` to `Virtualized.py`

Reply:
We replaced this with a `from_export` config, which is just used to tell us that this is from export and to skip over AOTAutograd. I plan to remove this flag within the next 2 weeks after fixing some other stuff on the export side. So is it ok to add it here for now?